hack/verify-kustomize.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+_BINDIR="$THIS_DIR/.bin"
+# Must be an exact match, as the generated YAMLs may not be consistent across
+# versions
+_KUSTOMIZE_VERSION="v4.4.1"
+
+# Ensure the kustomize tool exists and is the correct version, or installs it
+verify_kustomize() {
+    # Check if there is already a kustomize binary in $_BINDIR and if yes, check
+    # if the version matches the expected one.
+    local kustomize="$(PATH=$_BINDIR command -v kustomize)"
+    if [ -x "$kustomize" ]; then
+        # Verify version if kustomize was already installed.
+        local kustomize_version="$($kustomize version --short)"
+        # Should work with:
+        #  - kustomize/v3.3.0
+        #  - {kustomize/v3.8.2  2020-08-29T17:44:01Z  }
```

Comment:
I know this is the same as Antrea, but the comment does not have much sense if we're now using kustomize 4.4.1...

Reply:
Yep it's redundant. I'll remove the comment.